Microsoft.Alm.Authentication/TargetUri.cs

```diff
@@ -112,22 +107,71 @@ public bool IsBaseOf(TargetUri targetUri)
             if (targetUri == null)
                 return false;
 
-            return ResolvedUri.IsBaseOf(targetUri.ResolvedUri);
+            return ActualUri.IsBaseOf(targetUri.ActualUri);
         }
         /// <summary>
-        /// Gets a canonical string representation for the <see cref="ResolvedUri"/>.
+        /// Gets a canonical string representation for the <see cref="ActualUri"/>.
         /// </summary>
         /// <returns></returns>
         public override String ToString()
         {
-            return ResolvedUri.ToString();
+            return ActualUri.ToString();
+        }
+
+        public HttpClientHandler HttpClientHandler
+        {
+            get
+            {
+                bool useProxy = ProxyUri != null;
+                return new HttpClientHandler()
+                {
+                    Proxy = WebProxy,
+                    UseProxy = useProxy,
+                    MaxAutomaticRedirections = 2,
+                    UseDefaultCredentials = !useProxy
```

Comment:
I think that this line should be removed. Just because somebody is using a proxy, doesn't mean that they can't authenticate to the final endpoint using default credentials.